src/components/dialogs/load-creation-dialog.js

```diff
@@ -399,7 +404,12 @@ const LoadCreationDialog = ({ open, onClose, network }) => {
                             filterOptions={(options, params) => {
                                 const filtered = filter(options, params);
 
-                                if (params.inputValue !== '') {
+                                if (
+                                    params.inputValue !== '' &&
+                                    !options.find(
```

Comment:
`filtered.find` for optimization if we have a lot of options then the find could be greedy